libs/features/personalization/personalization.js

```diff
@@ -311,7 +311,7 @@ function parsePlaceholders(placeholders, config, selectedVariantName = '') {
 const checkForParamMatch = (paramStr) => {
   const [name, val] = paramStr.split('param-')[1].split('=');
   if (!name) return false;
-  const searchParamVal = PAGE_URL.searchParams.get(name);
+  const searchParamVal = PAGE_URL.searchParams.get(name).toLowerCase();
```

Comment:
Since searchParamVal can be `null`, let's safeguard the chained method.
```suggestion
  const searchParamVal = PAGE_URL.searchParams.get(name)?.toLowerCase();
```

Reply:
Agree with @mokimo, optional chaining should be used

Reply:
little embarrased I missed that.  Fixed.